src/inc/corinfo.h

```diff
@@ -1686,6 +1686,8 @@ struct CORINFO_CALL_INFO
     };
 
     CORINFO_CONST_LOOKUP    instParamLookup;    // Used by Ready-to-Run
+
+    BOOL                    secureInvoke;
```

Comment:
It is unfortunate to have to another 4 bytes in this structure for this rare case. We should either introduce a general purpose `DWORD flags;` field in `CORINFO_CALL_INFO`; or add the flag as global to `CORJIT_FLG2_XXXX` instead.

@dotnet/jit-contrib Opinions on which way we should be going?